README.md

```diff
@@ -3,34 +3,35 @@
 # Lab work <mark>NUMBER</mark>: <mark>SHORT TOPIC</mark>
 Authors (team): <mark>AUTHORS WITH GITHUB LINKS</mark><br>
 Variant: <mark>VARIANT SHOULD BE HERE</mark>
-## Before 
+## Prerequisites
 
-<mark>DESCRIBE PREREQUIREMENTS</mark>
+<mark>DESCRIBE PREREQUIREMENTS AS GCC OPENMP WHATEVER</mark>

```

Comment:
LIST LIBRARIES/TOOLS/OTHER UTILITIES THAT NEED TO BE INSTALLED (E.G. GCC, OPENMP, CMAKE ETC)